src/gui/search/searchtab.cpp

```diff
@@ -41,16 +42,26 @@
 
 #include "base/utils/misc.h"
 #include "base/preferences.h"
+#include "base/settingsstorage.h"
+#include "guiiconprovider.h"
 #include "searchsortmodel.h"
 #include "searchlistdelegate.h"
 #include "searchwidget.h"
 #include "searchtab.h"
 
+namespace
+{
+    constexpr char FILTER_MODE_SETTING_NAME[] = "Search/FilteringMode";
```

Comment:
If you indeed elect to save/load the other values too then please convert to the way we already do things. eg Look at the top of application.cpp

```c++
#define SETTINGS_KEY(name) "Search/" name

const QString KEY_FILTER_MODE_SETTING_NAME = SETTINGS_KEY("FilteringMode");
```

But in any case prefix this with `KEY_`